documentation/src/main/asciidoc/topics/proc_configuring_server_fips.adoc

```diff
@@ -0,0 +1,45 @@
+[id='configuring-server-fips_{context}']
+= Configuring {brandname} Server on a system with FIPS 140-2 compliant cryptography
+It is possible to configure {brandname} Server and configure it to present SSL/TLS certificates that verify its identity to clients.
```

Comment:
"It is possible to configure {brandname} Server to present SSL/TLS certificates that verify its identity to clients."

Reply:
Fixed